### PR TITLE
OpenCV Manager for Android (version 3)

### PR DESCRIPTION
**Pull request by mshabunin**
- removed android_camera
- removed standalone android package mode
- reworked manager package (renamed back to `org.opencv.engine`)

**Note:** there are some problems with native samples on x86_64 simulator

### CODE REVIEW

**Comment by vpisarev** [comment]
@alalek, please, review

**Comment by vpisarev** [comment]
@asmorkalov, I've put Alexander as main reviewer because we need to integrate this PR very quickly, on Wednesday. Still, if you have time now or later, please, take a look. We will fix the found issues as soon as we can.

File: modules/java/generator/src/java/android+CameraBridgeViewBase.java [hunk 1]
```diff
@@ -351,6 +356,7 @@ private void onExitStoppedState() {
     // NOTE: The order of bitmap constructor and camera connection is important for android 4.1.x
     // Bitmap must be constructed before surface
     private void onEnterStartedState() {
+        Log.d(TAG, "call ponEnterStartedState");
```
Comment (alalek) [inline modules/java/generator/src/java/android+CameraBridgeViewBase.java]: typo

File: modules/java/generator/src/java/android+AsyncServiceHelper.java [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import java.util.StringTokenizer;
 
 import org.opencv.core.Core;
-import org.opencv.engine3.OpenCVEngineInterface;
+import org.opencv.engine.OpenCVEngineInterface;
```
Comment (alalek) [inline modules/java/generator/src/java/android+AsyncServiceHelper.java]: `getPackageName` returns `"OpenCV3 Manager"` (twice). Is it OK?

**Comment by alalek** [comment]
:+1:

**Pull request merged**